### DISCUSSION TITLE
v1.0.1

### DISCUSSION THREAD

**Discussion by IndiMops** [IndiMops/translate-Red-Deat-Redemption-2#12] [Announcements]
### Вітаю друзі!
**Пройшло немало часу з попередньої публікації перекладу і ось ми презентуємо переклад другого розділу гри. Другий розділ гри презентує нам нові цікаві історії Артура Морґана та його банди в скотарському містечку Валентайн.**

## Що було додано:

- перекладено сюжетні завдання другого розділу(окрім деяких завдань, бо виникли деякі проблеми, найближчим часом додамо);
- перекладено інтерфейс для сюжетних завдань;
- перекладено радіальне меню зброї/предметів/коня;
- перекладено назви та описи предметів зброї;

## Було виправлено:
- виправили помилки в сюжетних завдань першого розділу;
- виправлено - #9;

### Встановлення
**Якщо ви встановлюєте вперше, то перейдіть [сюди](https://github.com/IndiMops/translate-Red-Deat-Redemption-2#%D1%8F%D0%BA-%D0%B2%D1%81%D1%82%D0%B0%D0%BD%D0%BE%D0%B2%D0%B8%D1%82%D0%B8-%D0%BB%D0%BE%D0%BA%D0%B0%D0%BB%D1%96%D0%B7%D0%B0%D1%86%D1%96%D1%8E). А якщо ви вже встановлювали локалізацію, то вам слід перенести файли з архіву в теку `Red Dead Redemption 2\lml\downloader\UA Localization For RDR 2` та підтвердити заміну файлів**

### Попередження
- переклад виконаний ентузіастами;
- переклад працює лише на ліцензійних версіях гри.
- перед запуском гри, увімкніть англійський мовний пакет в налаштуваннях платформи;

Дуже просимо, якщо знайдете помилки чи у вас є якісь пропозиції створюйте питання в [Issues](https://github.com/IndiMops/translate-Red-Deat-Redemption-2/issues)

<hr /><em>This discussion was created from the release <a href='https://github.com/IndiMops/translate-Red-Deat-Redemption-2/releases/tag/v1.0.1'>v1.0.1</a>.</em>

**Comment by KUT13** [comment]
Все ок, дякую за переклад, але щойно...
Підходжу до лікаря, дивлюсь шо можна продати шо купити, бачу "Цілитися", думаю зараз сцілюсь))) ну довелось перезагружатись з сейву)) лол
Ще секунд 10 думав чому це безкоштовно))
